src/pages/AcceptInvitation/index.tsx

```diff
@@ -1,30 +1,35 @@
+import { useMemo } from 'react'
 import { Navigate, useLocation } from 'react-router-dom'
 
 import { RoutePaths } from '@/enums'
 import { useNestedRoutes } from '@/hooks'
 
-import { EmailVerification, FillRequestForm } from './pages'
+import { EmailVerification, FillRequest } from './pages'
 
 export default function AcceptInvitation() {
   const location = useLocation()
 
+  const rootRedirectPath = useMemo(
+    () => `${RoutePaths.AcceptInvitationEmailVerification}${location.search}`,
+    [location.search],
+  )
+
   return useNestedRoutes(RoutePaths.AcceptInvitation, [
     {
       index: true,
-      element: (
-        <Navigate
-          replace
-          to={`${RoutePaths.AcceptInvitationEmailVerification}${location.search}`}
-        />
-      ),
+      element: <Navigate replace to={rootRedirectPath} />,
     },
     {
       path: RoutePaths.AcceptInvitationEmailVerification,
       element: <EmailVerification />,
     },
     {
       path: RoutePaths.AcceptInvitationFillRequest,
-      element: <FillRequestForm />,
+      element: <FillRequest />,
+    },
+    {
+      path: '*',
+      element: <Navigate replace to={rootRedirectPath} />,
```

Comment:
why do we need it?

Reply:
to be correctly redirected from /i/lol-kek-cheburek